electronics_lib/JLcResistor.py

```diff
@@ -27,11 +27,7 @@ def parse_row(row: PartsTableRow) -> Optional[Dict[PartsTableColumn, Any]]:
         new_cols: Dict[PartsTableColumn, Any] = {}
         try:
           # handle the footprint first since this is the most likely to filter
-          footprint = (cls.PACKAGE_FOOTPRINT_MAP.get(row['Package'], None))
-                       #or cls.SERIES_FOOTPRINT_MAP.get(row['Series'], None) or  TODO add later
-
-          if footprint is None:
-            raise KeyError
+          footprint = cls.PACKAGE_FOOTPRINT_MAP[row['Package']]
```

Comment:
You should be able to combine this with the next line. The inductor table didn't because it had more complex logic, but this is simple enough to not need a separate variable.